composer.json

```diff
@@ -36,7 +36,9 @@
         "phpunit/phpunit": "^9.5",
         "roave/infection-static-analysis-plugin": "^1.7",
         "spatie/phpunit-watcher": "^1.23",
-        "vimeo/psalm": "^4.3"
+        "vimeo/psalm": "^4.3",
+        "yiisoft/injector": "^1.0",
```

Comment:
This one should be in `require`, not `require-dev`.

Reply:
If you don't need Injector you don't need to install it.